README.md

```diff
@@ -15,33 +13,65 @@ This is the official WooCommerce extension to receive payments using the South A
 
-#### Does this require a Payfast merchant account?
+## Fast, super-secure, and scalable
 
-Yes! A Payfast merchant account, merchant key and merchant ID are required for this gateway to function.
+Pay it safe, every time. Payfast is trusted by **more than 100,000** South African businesses, with security that exceeds industry standards. It’s PCI DSS-compliant and also supports 18+ popular payment methods. In a nutshell, WooCommerce Payfast Gateway is the fastest, smartest, and safest way to accept payments online.
 
-#### Does this require an SSL certificate?
+## Boost your bottom line
 
-An SSL certificate is recommended for additional safety and security for your customers.
+Payfast is compatible with [WooCommerce Subscriptions](https://woocommerce.com/products/woocommerce-subscriptions/) and [WooCommerce Pre-Orders](https://woocommerce.com/products/woocommerce-pre-orders/), giving your shoppers the ultimate flexibility. **Convert more customers** by letting them make deposits, order in advance, or subscribe to your products and services. You win each way!
+
+## Frequently Asked Questions
 
 #### Where can I find documentation?
 
-For help setting up and configuring, please refer to our [user guide](https://docs.woocommerce.com/document/payfast-payment-gateway)
+You’ve come to the right place. [Our documentation](https://woocommerce.com/document/payfast-payment-gateway/) for WooCommerce Payfast Gateway includes detailed setup instructions.
+
+#### Where can I get support?
+
+Get in touch via the [official support forum](https://wordpress.org/support/plugin/woocommerce-payfast-gateway/).
+
+#### Does this extension support subscriptions?
+
+Yes! WooCommerce PayFast Gateway is compatible with [WooCommerce Subscriptions](https://woocommerce.com/products/woocommerce-subscriptions/).
+
+#### Do I need an SSL certificate?
+
+We recommend using a [Secure Sockets Layer (SSL) certificate](https://woocommerce.com/document/ssl-and-https/) for additional customer security and trust.
+
+#### Can I accept international payments?
+
+Yes! You can receive Visa and Mastercard payments from anywhere in the world.
+
+#### Do I need a Payfast account?
+
+Yes; a [Payfast merchant account](https://payfast.io/gateway-aggregator-selector/) is required.
+
+#### Do I need to be a South African resident to open a Payfast account?
+
+No, but to open a Payfast account, you’ll need a South African bank account to pay your funds into.
 
-#### Where can I get support or talk to other users?
 
-If you get stuck, you can ask for help in the Plugin Forum.
 
 ### Compatibility
 
 This extension is compatible with:
 - [WooCommerce Blocks](https://woo.com/products/woocommerce-gutenberg-products-block/)
+- [WooCommerce Subscriptions](https://woocommerce.com/products/woocommerce-subscriptions/)
+- [WooCommerce Deposits](https://woocommerce.com/products/woocommerce-deposits/)
```

Comment:
Above, this README mentions "Compatible with subscriptions, deposits, and pre-orders."

Should we add WC Pre-Orders here for completeness?

```suggestion
This extension is compatible with:
- [WooCommerce Blocks](https://woo.com/products/woocommerce-gutenberg-products-block/)
- [WooCommerce Subscriptions](https://woocommerce.com/products/woocommerce-subscriptions/)
- [WooCommerce Deposits](https://woocommerce.com/products/woocommerce-deposits/)
- [WooCommerce Pre-Orders](https://woocommerce.com/products/woocommerce-pre-orders/)
```

Reply:
Good spotting! Adding that in.